chrysom/listenerClient.go

```diff
@@ -131,15 +152,15 @@ func (c *ListenerClient) Start(ctx context.Context) error {
 				return
 			case <-c.observer.ticker.C:
 				outcome := SuccessOutcome
-				ctx := c.setLogger(context.Background(), c.logger)
+				ctx := c.setLogger(context.Background(), logger)
 				items, err := c.reader.GetItems(ctx, "")
 				if err == nil {
 					c.observer.listener.Update(items)
 				} else {
 					outcome = FailureOutcome
-					c.logger.Error("Failed to get items for listeners", zap.Error(err))
+					logger.Error("Failed to get items for listeners", zap.Error(err))
 				}
-				c.observer.measures.Polls.With(prometheus.Labels{
```

Comment:
Simplified how metrics are managed, `measures.Polls` can be directly accessed as `pollsTotalCounter`